content/installation/bootstrapping.md

```diff
@@ -124,35 +151,46 @@ To deploy Cloudify Manager using an image:
     {{< /gsHighlight >}}
 
  To access the Cloudify Manager UI, navigate to http://<_manager-ip_>/
+ 
+### What's Next
 
+You can now [upload a plugin]({{< relref "plugins/using-plugins.md" >}}) or [configure secrets]({{< relref "manager/using-secrets.md" >}}).
 
 ## Option 2 Bootstrapping a Cloudify Manager
 
-Bootstrapping consists of running a blueprint of the Cloudify Manager that installs and configures all of the Cloudify components. If you are installing Cloudify Manager in an offline environment, [click here]({{< relref "installation/bootstrapping.md#offline-environment" >}}) 
 
-#### Process Overview
+Bootstrapping consists of running a blueprint of the Cloudify Manager that installs and configures all of the Cloudify components. If you are installing Cloudify Manager in an offline environment, [click here]({{< relref "installation/bootstrapping.md#installing-cloudify-manager-in-an-offline-environment" >}}).
+
+You can install Cloudify using [pip 6.0 or higher](https://pip.pypa.io/en/stable/installing/). It is possible to operate Cloudify on [virtualenv 12.0 or higher](https://virtualenv.readthedocs.org/en/latest/installation.html). However, it is recommended that you download the Cloudify CLI package (see Step 1 of the following procedure).
```

Comment:
Since pip is not the main path, do we want to mention pip installation option as *NOTE* instead of under the main text?